modules/python3/CMakeLists.txt

```diff
@@ -1,168 +1,170 @@
+    include/modules/python3/processors/pythonscriptprocessor.h
+    include/modules/python3/pybindutils.h
+    include/modules/python3/python3module.h
+    include/modules/python3/python3moduledefine.h
+    include/modules/python3/pythonexecutionoutputobservable.h
+    include/modules/python3/pythoninterpreter.h
+    include/modules/python3/pythonlogger.h
+    include/modules/python3/pythonprocessorfactoryobject.h
+    include/modules/python3/pythonprocessorfolderobserver.h
+    include/modules/python3/pythonscript.h
+    include/modules/python3/pyutils.h
+)
+ivw_group("Header Files" ${HEADER_FILES})
+
+#--------------------------------------------------------------------
+# Add source files
+set(SOURCE_FILES
+    src/processors/numpymandelbrot.cpp
+    src/processors/numpymeshcreatetest.cpp
+    src/processors/numpyvolume.cpp
+    src/processors/pythonscriptprocessor.cpp
+    src/pybindutils.cpp
+    src/python3module.cpp
+    src/pythonexecutionoutputobservable.cpp
+    src/pythoninterpreter.cpp
+    src/pythonlogger.cpp
+    src/pythonprocessorfactoryobject.cpp
+    src/pythonprocessorfolderobserver.cpp
+    src/pythonscript.cpp
+    src/pyutils.cpp
+)
+ivw_group("Source Files" ${SOURCE_FILES})
+
+#--------------------------------------------------------------------
+# Add script files
+set(SCRIPT_FILES
+    scripts/documentgenerator.py
+    scripts/ivw/__init__.py
+    scripts/ivw/animation.py
+    scripts/ivw/camera.py
+    scripts/ivw/regression.py
+    scripts/ivw/utils.py
+
+    data/scripts/mandelbrot.py
+    data/scripts/numpymeshcreatetest.py
+    data/scripts/numpyvolumeprocessor.py
+    data/scripts/scriptprocessorexample.py
+)
+ivw_group("Script Files" ${SCRIPT_FILES})
+
+#--------------------------------------------------------------------
+# Add Unittests
+set(TEST_FILES
+    tests/unittests/python3-unittest-main.cpp
+    tests/unittests/scripts-test.cpp
+    tests/unittests/scripts/grabreturnvalue.py
+    tests/unittests/scripts/passvalues.py
+    tests/unittests/scripts/simple_buffer_test.py
+    tests/unittests/scripts/glm.py
+    tests/unittests/scripts/option_property.py
+)
+if(NOT NUMPY_OUTPUT_VERSION MATCHES "failed")
+    list(APPEND TEST_FILES tests/unittests/numpy-test.cpp)
+endif()
+ivw_add_unittest(${TEST_FILES})
+
+#--------------------------------------------------------------------
+# Create module
+ivw_create_module(${SOURCE_FILES} ${HEADER_FILES} ${SCRIPT_FILES})
+target_link_libraries(inviwo-module-python3 PUBLIC 
+    pybind11::embed 
+    $<BUILD_INTERFACE:${PYTHON_LIBRARIES}>
+)
+target_include_directories(inviwo-module-python3 PUBLIC
+    ${PYTHON_INCLUDE_DIR}
+    ${IVW_EXTENSIONS_DIR}/pybind11/include
+)
+
+add_subdirectory(bindings)
+
+if(IVW_UNITTESTS)
+    add_dependencies(inviwo-unittests-python3 inviwopy)
+endif()
+
+#--------------------------------------------------------------------
+# Add Python to package
+option(IVW_PACKAGE_EMBED_PYTHON "Embed a python dist in the installer.\


```

Comment:
What kind of end of line character is that (`^M`)

Reply:
windows?